src/components/metadata-form/index.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from "react"
+import { useFieldArray, useForm, useWatch } from "react-hook-form"
+import { Box, Flex, Text } from "rebass"
+import Button from "../button"
+import Card from "../card"
+import Input from "../input"
+import { get } from "lodash"
+
+// TODO: Filtering out null keys until Medusa Core update, remove this after
+const filterParentMetadata = parent => {
+  const filteredMetadata = Object.entries(parent.metadata).reduce(
+    (prev, [key, value]) => (value ? { ...prev, [key]: value } : prev),
```

Comment:
hmm, do we want filter out all falsey values, or only specifically null/undefined ones?

Reply:
This would remove false, 0, NaN, etc

Reply:
hmmmm I think you're right, gonna modify this to `value === null ? prev : {...prev, [key]: value}` because we shouldn't ever be getting falsey values submitted here. This is resolved!